cockatrice/src/dlg_settings.cpp

```diff
@@ -248,6 +268,7 @@ void GeneralSettingsPage::retranslateUi()
     replaysPathLabel.setText(tr("Replays directory:"));
     picsPathLabel.setText(tr("Pictures directory:"));
     cardDatabasePathLabel.setText(tr("Card database:"));
+    customCardDatabasePathLabel.setText(tr("Custom database directory:"));
```

Comment:
Can you swap ordering of the new option with the card database one?
That way we have directories on top, followed by file locations. While databases are still grouped together.

Also, we use the plural form for all others directories.

Reply:
I considered it ( I had it below replays first ) but I moved it down here because the main card database is more important, I'd consider moving it below tokens.
I made it singular because a database of databases is still a database, and with this now being the longest entry the place for long path names has been diminished for all entries :^)